dashboard/app/views/levels/_unplug.html.haml

```diff
@@ -1,5 +1,5 @@
 - unplugged = @level.type == 'Unplugged' ? @level.name : @game.name
-- video_key = try_t("data.unplugged.#{unplugged}.video")
+- video_key = @level.video_key || try_t("data.unplugged.#{unplugged}.video")
```

Comment:
...Although, maybe it's a feature to allow the video to vary by locale?  As far as I know we don't take advantage of this anywhere.  If we were to translate videos (vs. adding translated captions) I'd like to do this in a standardized way that also works for `video_key`.